tests/test_socket.py

```diff
@@ -2,6 +2,13 @@
 import pytest
 
 from pytest_socket import enable_socket
+from socket import gethostbyname
+
+real_http_ip_host = gethostbyname('httpbin.org')
```

Comment:
So the actual test suite makes network calls? Feels a bit counter to the ethos of the plugin, doesn't it?

Reply:
`test_urllib_succeeds_by_default` was already doing a real connection. Either way, I don't think it's against the ethos - to test that it works you have to check that it allows real network calls when disabled too. It's fair that we could start a simple http-server during the test, but `httpbin.org` was being used already, so I just tried to stay within the same approach.